jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/listeners/DatabaseSyncItemListener.java

```diff
@@ -23,33 +24,33 @@ public class DatabaseSyncItemListener extends ItemListener {
 
     @Inject
     public GlobalPipelineMavenConfig globalPipelineMavenConfig;
-    
+
     @Override
     public void onDeleted(Item item) {
-        if (item instanceof WorkflowJob) {
-            WorkflowJob pipeline = (WorkflowJob) item;
+        if (item instanceof BlockableResume) {
```

Comment:
Or just skip the `instanceof` or use `Job` since it would be harmless for other job types I guess.

For `WorkflowRun` there is a clear interface to use: `FlowExecutionOwner.Executable`